### PR TITLE
#333 - Fade unapproved change requests

### LINKED ISSUE #333: Proposed Solutions - Fade Unapproved Ones
### Description

After a change request has been approved, we want to fade the proposed solutions that weren't approved to make it clearer which one was.

### Acceptance Criteria

When a change request is approved (or denied), the not approved proposed solutions should have a lower opacity

### Proposed Solution

check out the code in the review modal for change requests because that fades the non selected ones so it should be similar

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by jaredlyon**
## Changes
Change requests that have not been approved appear less opaque as compared to approved requests - similarly to how unselected change requests are translucent in the review selection modal.

## Screenshots
Screen Shot 2023-02-08 at 8 06 29 PM

## To Do
- [x] Verify UI changes look good
- [x] Upload screenshots

## Checklist
- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket

Closes #333

### CODE REVIEW

File: src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionsList.tsx [hunk 1]
```diff
@@ -48,9 +48,10 @@ const ProposedSolutionsList: React.FC<ProposedSolutionsListProps> = ({ proposedS
   return (
     <>
       <div className={styles.proposedSolutionsList}>
-        {proposedSolutions.map((proposedSolution, i) => (
-          <ProposedSolutionView key={i} proposedSolution={proposedSolution} />
-        ))}
+        {proposedSolutions.map((proposedSolution, i) => {
```
Comment (anthonybernardi) [inline src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionsList.tsx:51]: i think the best way to do this would be to make `faded` a property of ProposedSolutionView and then pass it in as !proposedSolution.approved

then just need to implement something like this within the component itself

**Review by anthonybernardi — CHANGES_REQUESTED** [review]

File: src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionsList.tsx (hunk 1 above)
Comment (jaredlyon) [reply src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionsList.tsx:51]: [03afa94](https://github.com/Northeastern-Electric-Racing/FinishLine/pull/611/commits/03afa94e0155c05c8e786234b1faef8413b475f1) (maybe ????)

Comment (Ari-S-123) [inline src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionsList.tsx:51]: You should undo the changes in this file.

**Review by Ari-S-123 — CHANGES_REQUESTED** [review]

File: src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionsList.tsx (hunk 1 above)
Comment (jaredlyon) [reply src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionsList.tsx]: fixed in [c69b8ff](https://github.com/Northeastern-Electric-Racing/FinishLine/pull/611/commits/c69b8ff1dc462fd5bc0f367c08f0719ddd11cb0b)

**Review by anthonybernardi — CHANGES_REQUESTED** [review]
this fades proposed solutions even if the change request hasn't been reviewed yet which i don't think is what we want

**Review by anthonybernardi — APPROVED** [review]
awesome
